sklearn/utils/tests/test_validation.py

```diff
@@ -229,17 +229,19 @@ def test_check_array():
             assert_equal(X_checked.dtype, X.dtype)
         if order == 'C':
             assert X_checked.flags['C_CONTIGUOUS']
-            assert_false(X_checked.flags['F_CONTIGUOUS'])
+            assert not X_checked.flags['F_CONTIGUOUS']
         elif order == 'F':
             assert X_checked.flags['F_CONTIGUOUS']
-            assert_false(X_checked.flags['C_CONTIGUOUS'])
+            assert not X_checked.flags['C_CONTIGUOUS']
         if copy:
-            assert_false(X is X_checked)
+            assert X is not X_checked
         else:
             # doesn't copy if it was already good
-            if (X.dtype == X_checked.dtype and
-                    X_checked.flags['C_CONTIGUOUS'] == X.flags['C_CONTIGUOUS']
-                    and X_checked.flags['F_CONTIGUOUS'] == X.flags['F_CONTIGUOUS']):
+            if (
+                X.dtype == X_checked.dtype and
+                X_checked.flags['C_CONTIGUOUS'] == X.flags['C_CONTIGUOUS'] and
+                X_checked.flags['F_CONTIGUOUS'] == X.flags['F_CONTIGUOUS']
+            ):
```

Comment:
This is an unrelated change I think? I'm not convinced the new formatting is much better, better to revert this.